flask-hello-world/app.yaml

```diff
@@ -0,0 +1,5 @@
+command: [
+  "flask", "--app", 
```

Comment:
I feel like each argument should be on a new line, or --app should be with the app name.